tests/test_integration.py

```diff
@@ -203,9 +203,22 @@ def _activate_conda_script():
 
 def _environ_from_wrapper():
     python_wrapper = os.path.normpath("wrappers/python") + get_wrapper_extension()
-    output = subprocess.check_output(
-        [python_wrapper, "-c", "from os import environ; print(dict(environ))"]
-    )
+    try:
```

Comment:
Pytest provides a `pytest.raises` so you don't need to use try/except.
You could do something like:

```python
with raises(subprocess.CalledProcessError) as exc_info:
    output = subprocess.check_output(
        ...
        ...
    )

assert exc_info.value.returncode == 42